src/Forms/Prism.Forms/Ioc/ContainerRegistryExtensions.cs

```diff
@@ -6,7 +6,10 @@
 
 namespace Prism.Ioc
 {
-    public static class IContainerRegistryExtensions
+    /// <summary>
+    /// Extension methods for registering containers such as pages for navigation, dialog, etc.
```

Comment:
Extension methods for registering views and dialogs for navigation

Reply:
done